packages/flutter/lib/src/foundation/annotations.dart

```diff
@@ -0,0 +1,107 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+/// A category with which to annotate a class, for documentation
+/// purposes.
+///
+/// A category is usually represented as a section and a subsection, each
+/// of which is a string. The project to which the class belongs defines the
```

Comment:
"Authors" doesn't work. It makes it sound like whoever invents the class gets to invent the categories. Not sure how to make this clearer. I'll try "team"...